components/dashboard/src/workspaces/WorkspaceEntry.tsx

```diff
@@ -86,19 +86,31 @@ export function WorkspaceEntry({ desc, model, isAdmin, stopWorkspace }: Props) {
                 <WorkspaceStatusIndicator instance={desc?.latestInstance} />
             </div>
             <div className="flex flex-col w-3/12">
-                <a href={startUrl.toString()}><div className="font-medium text-gray-800 dark:text-gray-100 truncate hover:text-blue-600 dark:hover:text-blue-400">{ws.id}</div></a>
-                <a href={project ? 'https://' + project : undefined}><div className="text-sm overflow-ellipsis truncate text-gray-400 hover:text-blue-600 dark:hover:text-blue-400">{project || 'Unknown'}</div></a>
+                <Tooltip content={ws.id}>
+                    <a href={startUrl.toString()}>
+                        <div className="font-medium text-gray-800 dark:text-gray-100 overflow-ellipsis truncate w-full hover:text-blue-600 dark:hover:text-blue-400">{ws.id}</div>
+                    </a>
+                </Tooltip>
+                <Tooltip content={project || 'Unknown'}>
+                    <a href={project ? 'https://' + project : undefined}>
+                        <div className="text-sm overflow-ellipsis truncate w-full text-gray-400 hover:text-blue-600 dark:hover:text-blue-400">{project || 'Unknown'}</div>
+                    </a>
+                </Tooltip>
             </div>
-            <div className="flex w-4/12 truncate overflow-ellipsis">
-                <div className="flex flex-col">
-                    <div className="text-gray-500 overflow-ellipsis truncate">{ws.description}</div>
+            <div className="flex flex-col w-4/12">
+                <Tooltip content={ws.description}>
+                    <div className="text-gray-500 overflow-ellipsis truncate w-full">{ws.description}</div>
+                </Tooltip>
+                <Tooltip content={ws.contextURL}>
                     <a href={ws.contextURL}>
-                        <div className="text-sm text-gray-400 overflow-ellipsis truncate hover:text-blue-600 dark:hover:text-blue-400">{ws.contextURL}</div>
+                        <div className="text-sm text-gray-400 overflow-ellipsis truncate w-full hover:text-blue-600 dark:hover:text-blue-400">{ws.contextURL}</div>
                     </a>
-                </div>
+                </Tooltip>
             </div>
             <div className="flex flex-col items-start w-2/12">
-                <div className="text-gray-500 truncate">{currentBranch}</div>
+                <Tooltip content={currentBranch} className="w-full">
+                    <div className="text-gray-500 overflow-ellipsis truncate w-full">{currentBranch}</div>
```

Comment:
**suggestion**: May I suggest to leave the truncation on and remove most, if not all, tooltips? 🎈 

Ideally, we could make the _Context_ column so that it contains more compact information (see https://github.com/gitpod-io/gitpod/issues/3594). This way, we would only need a tooltip on the first row of the context column.

Additionally, on the _Pending Changes_ column we could truncate the branch name and offer a copy-to-clipboard button without also having a tooltip.